src/DUWOagent/predicates.pl

```diff
@@ -18,23 +18,23 @@
 
 % A predicate containing a building that doesn't influence our building indicators
 nonStudentBuilding(Bid,Name) :- 
-		buildings(Y),
-		self(OwnId),
-		member(building(Bid,Name,OwnId,Year,Cat,_,_,_),Y),
-		not(member('STUDENT',Cat)).
+	buildings(Y),
+	self(OwnId),
+	member(building(Bid,Name,OwnId,Year,Cat,_,_,_),Y),
+	not(member('STUDENT',Cat)).
 		
 getBuilding(Bid, Type) :- 
-		buildings(Y),
-		self(OwnId),
-		member(building(Bid,Name,OwnId,Year,Cat,_,_,_),Y),
-		member(Type, Cat).
+	buildings(Y),
+	self(OwnId),
+	member(building(Bid,Name,OwnId,Year,Cat,_,_,_),Y),
+	member(Type, Cat).
 
 % create a predicate with the Id's of all buildable houses of type student, so that the agent can dynamically choose what kind of building to build
 buildableStudentList([]).
 
 buildableStudent(Id,Name) :- functions(FS), 
```

Comment:
Maybe put `functions(FS)` also on a new line? Also fix the short names in this file as well.